types/brotli/decompress.d.ts

```diff
@@ -0,0 +1,5 @@
+/// <reference types="node" />
+
+declare function decompress(buffer: Buffer, outputSize?: number): Uint8Array;
```

Comment:
```suggestion
declare function decompress(buffer: Buffer, outputSize?: number): Uint8Array;
```
it's enough to reference in the `index.d.ts` (I'll add required suggestion snippet).
Simpler to maintain.